register/templates/apply.html

```diff
@@ -3,9 +3,11 @@
 {% block head_title %}Apply{% endblock %}
 {% block body %}
     {% settings_value 'REGISTER_APP' as app_data %}
-    {% if user.hacker %}
+    {% if not applications_open %}
+        <h4 class="text-center margin-top" style="color: black;">Applications are closed. We are looking forward to welcome you next edition.</h4>
```

Comment:
Applications are closed. We are looking forward to welcome you `on the` next edition.